SSA/Projects/InstCombine/HackersDelight.lean

```diff
@@ -0,0 +1,83 @@
+import Init.Data.BitVec.Basic
+import Init.Data.BitVec.Lemmas
+import Init.Data.BitVec.Bitblast
+
+namespace HackersDelight
+
+namespace Ch2Basics
+
+-- simple x
+theorem neg_eq_not_add_one {x : BitVec w} : - x = ~~~ x + 1 := by
+  apply BitVec.neg_eq_not_add
```

Comment:
```suggestion
theorem neg_eq_not_add_one {x : BitVec w} :
    - x = ~~~ x + 1 := by
  apply BitVec.neg_eq_not_add
```

I suggest the following formatting, as it will make it easier to match the formulas, as they are on a new line always 4 columns indented.

Reply:
Also, I feel there is no whitespace between a character and the unary `-`?